docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -e
+
+### Usage: /docker-entrypoint.sh <command> <args>
+command=$1
+
+case "$command" in
+  serve)
+    for file_candidate in $@ ; do
+      if test -f "$file_candidate"; then
+        tmpfile=$(mktemp /tmp/dex.config.yaml-XXXXXX)
+        gomplate -f "$file_candidate" -o "$tmpfile"
+
+        args="${args} ${tmpfile}"
+      else
+        args="${args} ${file_candidate}"
+      fi
+    done
+    exec dex $args
+    ;;
+  --help|-h|version)
+    exec dex $@
+    ;;
+  *)
+    exec $@
```

Comment:
I like this option, allowing it bypass the templating by adding `dex` in front of the arguments. 👍 
Maybe mention it in the "Usage" comment, so it's more obvious.

Reply:
You should also adapt the section "Does this PR introduce a user-facing change?":

* By default, Gomplate is now applied on the config file for command `serve ...`.
* To opt out, one can either adapt the command `serve ...` -> `dex serve ...`, or override the entrypoint.

Reply:
Yes, you are right! Whole description has to be rewritten.